### DISCUSSION TITLE
Can't install firedrake due to PETSc related problem

### DISCUSSION THREAD

**Discussion by xw719** [firedrakeproject/firedrake#2625] [Firedrake support]
Hi,

I was downloading firedrake with the instruction on the website, but it failed when configuring the PETSc. I am not an expert on computer and wonder if anyone could give me some suggestions. Please find the installation log below.

[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/9979566/firedrake-install.log)

Thanks in advance!

**Comment by connorjward** [comment]
Please can you share the `configure.log` from `firedrake/src/petsc`?

**Reply by xw719** [reply]
Thanks for your reply. Please find the `configure.log` here.
[configure.log](https://github.com/firedrakeproject/firedrake/files/9979865/configure.log)

**Reply by xw719** [reply]
Hi, I just upgraded PETSC and tried to install firedrake again. There was still some error. Please find the new `firedrake-install.log` and `configure.log` below. Thank you very much!
[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/9979913/firedrake-install.log)
[configure.log](https://github.com/firedrakeproject/firedrake/files/9979916/configure.log)

**Reply by connorjward** [reply]
What do you mean you upgraded PETSc? It should be installed as part of `firedrake-install`. You shouldn't be able to modify it.

The new error you are getting is a known issue:

> I see a lot of warnings like
> `ld: warning: object file (/U6sers/rdm4317/Development/firedrake/src/petsc/default/lib/libscotch.a(wgraph_store.o)) was built for newer macOS version (11.0) than being linked (10.15)`
> Can you modify the line in `firedrake-install`
> `with environment(MACOSX_DEPLOYMENT_TARGET="10.15"):`
> to
> `with environment(MACOSX_DEPLOYMENT_TARGET="11.0"):`
> and see if this resolves things.
